python/jupytergis_lab/jupytergis_lab/notebook/geo_debug.py

```diff
@@ -0,0 +1,33 @@
+from sidecar import Sidecar
+
+from jupytergis_lab import GISDocument
+
+
+def geo_debug(geojson_path: str) -> None:
```

Comment:
Trying to think of a prettier name, how about as simple as "show"?

```suggestion
def show(geojson_path: str) -> None:
```

Reply:
We settled on `explore` after some discussion at a community meeting!